extensions/objects/templates/svg-filename-editor.component.ts

```diff
@@ -407,6 +410,84 @@ angular.module('oppia').component('svgFilenameEditor', {
         };
       };
 
+      var loadGroupedObject = function(objId, obj, groupedObjects) {
+        // Checks if the id starts with 'group' to identify whether the
+        // svg objects are grouped together.
+        if (objId.startsWith('group')) {
+          // The objId is of the form "group" + number.
+          var groupId = parseInt(objId.slice(5));
+          // Checks whether the object belongs to an already existing group
+          // or not.
+          if (groupedObjects.length <= groupId) {
+            groupedObjects.push([]);
+          }
+          obj.toSVG = ctrl.createCustomToSVG(
+            obj.toSVG, obj.type, obj.id);
+          groupedObjects[groupId].push(obj);
+        }
+        return groupedObjects;
+      };
+
+      var loadTextObject = function(element, obj) {
+        var childNodes = [].slice.call(element.childNodes);
+        var value = '';
+        var coloredTextIndex = [];
+        // It extracts the text from the tspan tags and appends
```

Comment:
What does "it" in this sentence refer to (https://developers.google.com/tech-writing/one/words#it_and_they)? You can just omit "it".

Reply:
Done.